src/App.js

```diff
@@ -36,6 +37,24 @@ function App() {
   const [ingredients, setIngredients] = useState([{ name: "", amount: "" }]);
   const [directions, setDirections] = useState([""]);
 
+  const lambdaUrl = 'https://clv4h2vv73wgkhwucmru2cocqi0ifdre.lambda-url.us-east-1.on.aws/';
```

Comment:
We need to remove this from git and make sure its also removed from the history. Esp since this does not require any auth